### DISCUSSION TITLE
How to judge whether a linestring is on the left or right of another linestring？

### DISCUSSION THREAD

**Discussion by pucx** [NetTopologySuite/NetTopologySuite#617] [General]
In NTS，Is there an existing method to  judge whether a linestring is on the left or right of another linestring？thanks.

**Comment by DGuidi** [comment]
Maybe you can use the [LineSegment](https://github.com/NetTopologySuite/NetTopologySuite/blob/007103657e571c95900e3897b357a226efe65059/src/NetTopologySuite/Geometries/LineSegment.cs#L162) class?
